test/test_helper.rb

```diff
@@ -66,6 +67,7 @@ class Commenter < ActiveRecord::Base
   include DestroyedAt
   has_many :comments, dependent: :destroy
   has_many :posts, through: :comments
+  has_one :avatar, dependent: :destroy
```

Comment:
No need to add this relationship to the Commenter, as the Author already has this same relationship with the avatar. Thus, the 'destroys a has_one child correctly with dependent destroy' test will be duplicating the 'destroys child when parent does not mixin DestroyedAt' test that already exists.